configure.ac

```diff
@@ -36,10 +36,10 @@ AM_SILENT_RULES([yes])
 # 3. if interfaces were removed, then use C+1:0:0
 ##########################################################################
 
-ODP_LIBSO_VERSION=117:0:0
+ODP_LIBSO_VERSION=118:0:0
 AC_SUBST(ODP_LIBSO_VERSION)
 
-ODPHELPER_LIBSO_VERSION=113:2:1
+ODPHELPER_LIBSO_VERSION=118:0:0
```

Comment:
Helper is compatible, as far as I understand, so we should not bump major version here.

Reply:
Perhaps, but as this is a new LTS release might be best to sync everything? Apps are going to be recompiling for Tiger Moth anyway if they're coming from Monarch.

Reply:
@Bill-Fischofer-Linaro soname is not about releases, but about compatibility, so I vote for 114:0:2 for odphelper.